### PR TITLE
Removed unecessary semicolon

### PR DESCRIPTION
**Pull request by carloscaldas**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/684)

### CODE REVIEW

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

File: src/main/scala/beam/agentsim/agents/ridehail/allocation/EVFleetAllocationManager.scala [hunk 1]
```diff
@@ -63,8 +63,7 @@ class EVFleetAllocationManager(val rideHailManager: RideHailManager)
               .remainingRangeInM < routingResponses.get
               .map(
                 _.itineraries
-                  .filter(_.tripClassifier == RIDE_HAIL)
```
Comment (JustinPihony) [inline src/main/scala/beam/agentsim/agents/ridehail/allocation/EVFleetAllocationManager.scala]: I believe you have this in another PR - and rightly so, per the name of this PR

**Comment by carloscaldas** [comment]
@JustinPihony change addressed

**Pull request merged**